### PR TITLE
Swift 2.0

### PR DESCRIPTION
**Pull request by madbat**
I've done the minimum stuff to make Set Swift 2.0-ready.

Let me know if there's anything else I should do :)

### CODE REVIEW

**Comment by robrix** [comment]
Thanks for the PR!

File: SetTests/MultisetPrintableTests.swift [hunk 1]
```diff
@@ -19,14 +13,10 @@ final class TestElement: NSObject, Printable, DebugPrintable {
 }
 
 final class MultisetPrintableTests: XCTestCase {
-	func testDescription() {
-		XCTAssertEqual(Multiset<Int>().description, "{}")
-		let test = Multiset<TestElement>()
-		XCTAssertEqual(Multiset(TestElement()).description, "{description}")
-	}
```
Comment (robrix) [inline SetTests/MultisetPrintableTests.swift]: How come this test is being removed?

Comment (madbat) [inline SetTests/MultisetPrintableTests.swift]: My bad! I had deleted it while trying to understand compilation errors, and committed without it. 😓
I've already updated the PR, reintroducing it.

**Comment by robrix** [comment]
Thank you! :+1:

File: SetTests/MultisetPrintableTests.swift (hunk 1 above)
Comment (robrix) [inline SetTests/MultisetPrintableTests.swift]: Not to worry, these things happen :smile:

**Comment by madbat** [comment]
🎉

**Pull request merged**